examples/embeddable_explorer/public/list_container_example.tsx

```diff
@@ -0,0 +1,177 @@
+      '2': {
+        type: TODO_EMBEDDABLE,
+        explicitInput: {
+          id: '2',
+          task: 'Goes out on Wenesdays!',
+          icon: 'broom',
+          title: 'Take out the trash',
+        },
+      },
+      '3': {
+        type: TODO_EMBEDDABLE,
+        explicitInput: {
+          id: '3',
+          icon: 'broom',
+          title: 'Vaccum the floor',
+        },
+      },
+    },
+  };
+
+  const searchableInput = {
+    id: '1',
+    title: 'My searchable todo list',
+    panels: {
+      '1': {
+        type: HELLO_WORLD_EMBEDDABLE,
+        explicitInput: {
+          id: '1',
+          title: 'Hello',
+        },
+      },
+      '2': {
+        type: TODO_EMBEDDABLE,
+        explicitInput: {
+          id: '2',
+          task: 'Goes out on Wenesdays!',
+          icon: 'broom',
+          title: 'Take out the trash',
+        },
+      },
+      '3': {
+        type: MULTI_TASK_TODO_EMBEDDABLE,
+        explicitInput: {
+          id: '3',
+          icon: 'searchProfilerApp',
+          title: 'Learn more',
+          tasks: ['Go to school', 'Watch planet earth', 'Read the encylopedia'],
+        },
+      },
+    },
+  };
+
+  return (
+    <EuiPageBody>
+      <EuiPageHeader>
+        <EuiPageHeaderSection>
+          <EuiTitle size="l">
+            <h1>List container example</h1>
+          </EuiTitle>
+        </EuiPageHeaderSection>
+      </EuiPageHeader>
+      <EuiPageContent>
+        <EuiPageContentBody>
+          <EuiText>
+            Here is a container embeddable that contains other embeddables and displays them in a
+            list
+          </EuiText>
+          <EuiPanel data-test-subj="listContainerEmbeddablePanel" paddingSize="none" role="figure">
+            <EmbeddableFactoryRenderer
+              getEmbeddableFactory={getEmbeddableFactory}
+              type={LIST_CONTAINER}
+              input={listInput}
+            />
+          </EuiPanel>
+
+          <EuiSpacer />
+          <EuiText>
+            <p>
+              The reason to use a container embeddable instead of just a custom react component is
+              because it comes with helpful methods to store the state of all it&#39;s children
+              embeddables, listeners to clean up the input state when an embeddable is added or
+              removed, and a way to pass down the container embeddable&#39;s own input to it&#39;s
+              children. For this particular example, the container does not take any input.
+              Let&#39;s modify it so it does.
+            </p>
+            <p>
+              Now in this second example, the container takes in a filter string as input and passes
+              that down to all it&#39;s children.
+            </p>
+          </EuiText>
+
+          <EuiSpacer />
+          <EuiPanel
+            data-test-subj="searchableListContainerEmbeddablePanel"
+            paddingSize="none"
+            role="figure"
+          >
+            <EmbeddableFactoryRenderer
```

Comment:
I find the fact that `filter` applies only to inner items to be confusing
If you could hide empty items or gray them out, it would be much clearer.

Reply:
Yea, I agree, but I wanted an example of a container passing data down to the child.  I agree the scenario you mention would make more sense but we'd need them to implement something so the child could communicate "i have a match vs no match" to it's parent and we don't have that kind of communication yet.  

Think of it like on the dashboard - we pass down the filters and the child does something with that, but the container never changes how the child is rendered.

It is something to think about though.  I wonder if there is a better example I could do here.  Maybe pass down "highlight" and highlight the search terms instead of filter by them?  What do you think?

Reply:
Actually that's a great idea - search highlight would make it much clearer